app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,261 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
```

Comment:
- [ ] Kindly ensure that your design matches the given design by following the colors, font face, size, and weight as this is a requirement

Your design
![image](https://github.com/beteland123/smart_budget/assets/81932478/8e4f31ba-a93a-4c8b-ab34-e1db263a0274)

Given design
![image](https://github.com/beteland123/smart_budget/assets/81932478/c12c6861-8e16-43fc-8d25-5087e595e249)

- there is an error in the console on my end that is preventing the font from working if that not the case the next reviewer will ignore this comment
![image](https://github.com/beteland123/smart_budget/assets/81932478/7ed24c4f-6012-41c3-8a8e-140363d9178a)

- pay close attention to the font weight for the text as this will change the feel of the application to match the given design more for the transaction page

Your design
![image](https://github.com/beteland123/smart_budget/assets/81932478/6edb3d0c-4892-4273-a91e-e96af20f760f)


Given design
![image](https://github.com/beteland123/smart_budget/assets/81932478/a37df19f-a483-4401-ad91-14a0bd24e14b)

- Notice the difference in design, kindly fix this
- Consider making the sign-out button a white link and consider pushing the `Payments Page` heading to the blue top part as is in the given design and changing it to a white color
#### Note that this should apply for the categories page